src/components/application_manager/test/commands/hmi/get_urls_test.cc

```diff
@@ -97,12 +98,34 @@ class GetUrlsTest : public CommandsTest<CommandsTestMocks::kIsNice> {
         utils::MakeShared<policy_manager_test::MockPolicyManager>();
     ASSERT_TRUE(mock_policy_manager_);
   }
+
+  MessageSharedPtr CreateCommandMsg() {
+    MessageSharedPtr command_msg(
+        CreateMessage(NsSmartDeviceLink::NsSmartObjects::SmartType_Map));
+    (*command_msg)[am::strings::params][am::strings::connection_key] =
+        kConnectionKey;
+    (*command_msg)[am::strings::msg_params][am::hmi_request::service] =
+        kPolicyService;
+
+    return command_msg;
+  }
+
+  RequestFromHMIPtr PrepareRunFlowAndCreateCommandRequestFromHMI(
+      MessageSharedPtr command_msg) {
+    ON_CALL(mock_app_manager_, event_dispatcher())
```

Comment:
@dcherniev, please derive `GetUrlsTest` from `CommandRequestTest` 
It provide `ON_CALL` for `event_dispatcher` and `mock_event_dispatcher_`

-
Also, if you will set command and message as a members of `GetUrlsTest`, you can move all actions form `CreateCommandMsg()` into constructor.